<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeWallet</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabExpenses">
       <attribute name="title">
        <string>Expenses</string>
       </attribute>
       <layout class="QVBoxLayout" name="layExpenses">
        <item>
         <widget class="QTableView" name="tvExpenses">
          <property name="font">
           <font>
            <family>Verdana</family>
           </font>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabIncomes">
       <attribute name="title">
        <string>Incomes</string>
       </attribute>
       <layout class="QVBoxLayout" name="layIncomes">
        <item>
         <widget class="QTableView" name="tvIncomes">
          <property name="font">
           <font>
            <family>Verdana</family>
           </font>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layControls">
      <item>
       <widget class="QGroupBox" name="gbQuickFilter">
        <property name="maximumSize">
         <size>
          <width>512</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Quick filter</string>
        </property>
        <layout class="QHBoxLayout" name="layQuickFilter">
         <item>
          <widget class="QLabel" name="lbQuickFilter">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leQuickFilter"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Quick_Filter_Apply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbFilter">
        <property name="title">
         <string>Filter</string>
        </property>
        <layout class="QGridLayout" name="layFilter">
         <item row="0" column="0">
          <widget class="QCheckBox" name="cbDateFrom">
           <property name="text">
            <string>Date from</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="dteDateFrom"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cbDateTo">
           <property name="text">
            <string>Date to</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="dteDateTo"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lbCat">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="lbSubcat">
           <property name="text">
            <string>Subcategory</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="cbCategory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="cbSubcategory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="btn_Filter_Apply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="btn_Filter_Reset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Add">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add...</string>
        </property>
        <property name="icon">
         <iconset resource="homewallet.qrc">
          <normaloff>:/img/32x32/add_32x32.png</normaloff>:/img/32x32/add_32x32.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Edit">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Edit...</string>
        </property>
        <property name="icon">
         <iconset resource="homewallet.qrc">
          <normaloff>:/img/32x32/edit_32x32.png</normaloff>:/img/32x32/edit_32x32.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Delete">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Delete...</string>
        </property>
        <property name="icon">
         <iconset resource="homewallet.qrc">
          <normaloff>:/img/32x32/remove_32x32.png</normaloff>:/img/32x32/remove_32x32.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Import"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Categories"/>
    <addaction name="separator"/>
    <addaction name="actionFilter"/>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="action_DbDebug"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Import">
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Categories">
   <property name="text">
    <string>&amp;Categories</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_DbDebug">
   <property name="text">
    <string>&amp;DB debug</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>&amp;Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="homewallet.qrc"/>
 </resources>
 <connections/>
</ui>
